### PR TITLE
13.0 Change prices

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: pos_orders_history_return/__manifest__.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_orders_history_return/",
     "license": "LGPL-3",
-    "price": 20.00,
+    "price": 10.00,
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_return/__manifest__.py]: Price = 20.0. The price was changed before.

File: pos_cashier_select/__manifest__.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_cashier_select/",
     "license": "LGPL-3",
-    "price": 39.00,
+    "price": 19.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_cashier_select/__manifest__.py]: price = 20.0

File: pos_product_available_negative/__manifest__.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_product_available_negative/",
     "license": "LGPL-3",
-    "price": 25.00,
+    "price": 12.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_product_available_negative/__manifest__.py]: price = 25.0 (already changed)

File: pos_order_printer_product/__manifest__.py [hunk 4]
```diff
@@ -11,7 +11,7 @@
     "support": "pos@it-projects.info",
     "website": "https://twitter.com/gabbasov_dinar",
     "license": "LGPL-3",
-    "price": 49.00,
+    "price": 24.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_order_printer_product/__manifest__.py]: price = 25.0

File: pos_print_method/__manifest__.py [hunk 5]
```diff
@@ -11,7 +11,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_print_method/",
     "license": "LGPL-3",
-    "price": 17.00,
+    "price": 8.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_print_method/__manifest__.py]: 17.0

File: pos_cashbox/__manifest__.py [hunk 6]
```diff
@@ -13,7 +13,7 @@
     "support": "pos@it-projects.info",
     "website": "https://it-projects.info/team/GabbasovDinar",
     "license": "LGPL-3",
-    "price": 49.00,
+    "price": 24.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_cashbox/__manifest__.py]: 25

File: pos_pin/__manifest__.py [hunk 7]
```diff
@@ -16,7 +16,7 @@
         'data.xml',
     ],
 
-    "price": 10.00,
+    "price": 5.00,
```
Comment (Ramil-Mukhametzyanov) [inline pos_pin/__manifest__.py]: 10

File: pos_logout/__manifest__.py [hunk 8]
```diff
@@ -15,7 +15,7 @@
     "support": "pos@it-projects.info",
     "website": "https://it-projects.info/team/GabbasovDinar",
     "license": "LGPL-3",
-    "price": 39.00,
+    "price": 19.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_logout/__manifest__.py]: 20

File: pos_orderline_absolute_discount/__manifest__.py [hunk 9]
```diff
@@ -11,7 +11,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_orderline_absolute_discount/",
     "license": "LGPL-3",
-    "price": 40.00,
+    "price": 20.00,
```
Comment (Ramil-Mukhametzyanov) [inline pos_orderline_absolute_discount/__manifest__.py]: 40

File: pos_partner_deselection/__manifest__.py [hunk 10]
```diff
@@ -14,7 +14,7 @@
     "support": "apps@it-projects.info",
     "website": "https://it-projects.info/team/KolushovAlexandr",
     "license": "LGPL-3",
-    "price": 15.00,
+    "price": 7.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_partner_deselection/__manifest__.py]: 15

File: pos_partner_sync/__manifest__.py [hunk 11]
```diff
@@ -14,7 +14,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_barcode_sync/",
     "license": "LGPL-3",
-    "price": 25.00,
+    "price": 12.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_partner_sync/__manifest__.py]: 25

File: pos_qr_payments/__manifest__.py [hunk 12]
```diff
@@ -13,7 +13,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_qr_payments/",
     "license": "LGPL-3",
-    "price": 10.00,
+    "price": 5.00,
```
Comment (Ramil-Mukhametzyanov) [inline pos_qr_payments/__manifest__.py]: 10

File: pos_qr_show/__manifest__.py [hunk 13]
```diff
@@ -13,7 +13,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_qr_show/",
     "license": "LGPL-3",
-    "price": 20.00,
+    "price": 10.00,
```
Comment (Ramil-Mukhametzyanov) [inline pos_qr_show/__manifest__.py]: 20

File: pos_wechat/__manifest__.py [hunk 14]
```diff
@@ -13,7 +13,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_payment_wechat/",
     "license": "LGPL-3",
-    "price": 165.00,
+    "price": 82.50,
```
Comment (Ramil-Mukhametzyanov) [inline pos_wechat/__manifest__.py]: 165

File: qr_payments/__manifest__.py [hunk 15]
```diff
@@ -13,7 +13,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/pos_qr_payments/",
     "license": "LGPL-3",
-    "price": 10.00,
+    "price": 5.00,
```
Comment (Ramil-Mukhametzyanov) [inline qr_payments/__manifest__.py]: 10

File: wechat/__manifest__.py [hunk 16]
```diff
@@ -14,7 +14,7 @@
     "support": "pos@it-projects.info",
     "website": "https://apps.odoo.com/apps/modules/12.0/wechat/",
     "license": "LGPL-3",
-    "price": 75.00,
+    "price": 37.50,
```
Comment (Ramil-Mukhametzyanov) [inline wechat/__manifest__.py]: 75

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by ilmir-k** [comment]
I approve to merge it now

**Pull request merged**